var/spack/repos/builtin/packages/intel/package.py

```diff
@@ -0,0 +1,113 @@
+
+        components = base_components
+        if not spec.satisfies('+all'):
+            if spec.satisfies('+mpi') and 'cluster' in str(spec.version):
+                components += ";" + mpi_components
+            if spec.satisfies('+mkl'):
+                components += ";" + mkl_components
+            if spec.satisfies('+daal'):
+                components += ";" + daal_components
+            if spec.satisfies('+ipp'):
+                components += ";" + ipp_components
+            if spec.satisfies('+tools') and (spec.satisfies('@cluster') or spec.satisfies('@professional')):
+                components += ";" + tools_components
+
+        with open('install.ini', 'w') as f:
+            f.write("""
+ACCEPT_EULA=accept
+PSET_MODE=install
+CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
+PSET_INSTALL_DIR=%s
+ACTIVATION_LICENSE_FILE=%s
+ACTIVATION_TYPE=license_file
+PHONEHOME_SEND_USAGE_DATA=no
+COMPONENTS=%s
+""" %(prefix, license_path, components))
+
+        install_script = which("install.sh")
+
+        install_script('--silent', 'install.ini')
+
+        absbindir = os.path.split(os.path.realpath(os.path.join(self.prefix.bin, "icc")))[0]
+        abslibdir = os.path.split(os.path.realpath(os.path.join(self.prefix.lib, "intel64", "libimf.a")))[0]
+
+        # symlink or copy?
+        os.symlink(license_path, os.path.join(absbindir, "license.lic"))
+        if spec.satisfies('+tools') and (spec.satisfies('@cluster') or spec.satisfies('@professional')):
+            os.mkdir(os.path.join(self.prefix, "inspector_xe/licenses"))
+            os.symlink(license_path, os.path.join(self.prefix, "inspector_xe/licenses", "license.lic"))
+            os.mkdir(os.path.join(self.prefix, "advisor_xe/licenses"))
+            os.symlink(license_path, os.path.join(self.prefix, "advisor_xe/licenses", "license.lic"))
+            os.mkdir(os.path.join(self.prefix, "vtune_amplifier_xe/licenses"))
+            os.symlink(license_path, os.path.join(self.prefix, "vtune_amplifier_xe/licenses", "license.lic"))
```

Comment:
Yeah, annoyingly, each component looks in a different place for licenses, so a list would be helpful.